<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AI工具箱</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/AI Studio.ico</normaloff>:/images/AI Studio.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_main" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_main{background:#1F1F1F;border-radius:2px;}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,3" columnstretch="0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_top" native="true">
         <property name="minimumSize">
          <size>
           <width>800</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_logo">
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/header-logo.png</normaloff>:/images/header-logo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>191</width>
              <height>51</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_project">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>项目</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dataset">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>数据集</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_course">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>课程</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_competition">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>比赛</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_auth">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>认证</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_community">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>社区</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ppweb">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>点击访问</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background:Transparent;text-align: left;color:white;}QPushButton:hover{color:rgba(255,255,255, 0.8);}</string>
            </property>
            <property name="text">
             <string>访问飞桨官网</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_down" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#widget_down{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_left" native="true">
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>25</y>
               <width>141</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>图像类工具</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>651</width>
               <height>221</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0,1,0,1,0">
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>动物分类</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_2">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_2{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>中文识别</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_3">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_3{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>目标检测</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_4">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_4{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>图像风格迁移</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_13">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0,1,0,1,0">
                <item>
                 <spacer name="horizontalSpacer_14">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_5">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_5{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>秃头生成器</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_15">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_6">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_6{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>口罩识别</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_16">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_7">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_7{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>老照片上色</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_17">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="image_generation_8">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>14</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="toolTip">
                   <string>点击使用</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton#image_generation_8{background:#F0F5FF;border-bottom-left-radius:2px;border-bottom-right-radius:2px;}</string>
                  </property>
                  <property name="text">
                   <string>图像分割</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_18">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
